R/model_importance.R

```diff
@@ -47,18 +47,32 @@
 #' @export
 #'
 #' @examples
-mod_imp <- function(forecast_data,
-                    true_value,
-                    ensemble_fun = c("simple_ensemble", "linear_pool"),
-                    agg_fun = "mean",
-                    weighted = FALSE,
-                    training_window_length = 0,
-                    importance_algorithm = c("lomo", "lasomo"),
-                    subset_wt = c("equal", "perm_based"),
-                    scoring_rule = c("MAE", "MSE", "WIS", "CRPS", "Logscore"),
-                    na_action = c("worst", "average", "drop")) {
+model_importance <- function(forecast_data,
```

Comment:
maybe add a few tests around data formats, e.g.

- can forecast_data be read as a model_out_tbl
- is true_value in the correct format?
- ...

Reply:
https://github.com/orgs/hubverse-org/discussions/9#discussioncomment-11054549